test/integration/apiserver/apiserver_test.go

```diff
@@ -1957,16 +1963,18 @@ func TestTransform(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			rv, _ := strconv.Atoi(obj.GetResourceVersion())
-			if rv < 1 {
-				rv = 1
+			var rv string
+			if obj.GetResourceVersion() == "" {
+				rv = "0"
```

Comment:
zero has a special meaning. Why is that good here?

Also, please add a go doc about the intent.

Reply:
This is existing behavior, I'm not changing it. I can add a comment to clarify, but ultimately this was how the test authors decided to pass the information around.